### DISCUSSION TITLE
Versioning of future branches and tags?

### DISCUSSION THREAD

**Discussion by eitsupi** [quarto-dev/quarto-cli#1100] [Q&A] [answered]
There is currently a branch named `v1.1`, which is confusing with the version of quarto itself, so it may be better to remove it.
(I think it is common practice to create a minor version branch and manage patch versions with tags.)

https://github.com/quarto-dev/quarto-cli/tree/v1.1
https://github.com/quarto-dev/quarto-cli/commit/101b4a46fc0ec3934824c2911105bbfa06bbd505

**Comment by jjallaire** [comment, marked as answer]
Good point! Just removed that branch.
